shared/devices/render.desktop.js

```diff
@@ -60,7 +60,7 @@ const DeviceRow = ({device, revoked, showRemoveDevicePage, showExistingDevicePag
       <Box style={revoked ? stylesRevokedIconColumn : stylesIconColumn}>
         <Icon type={icon} />
       </Box>
-      <Box style={stylesCommonColumn}>
+      <Box style={stylesCommonColumn} onClick={() => showExistingDevicePage(device)}>
```

Comment:
The type for this function is wrong in the flow type def, (the type def takes no arguments)